test/spec/LoginRouter_spec.js

```diff
@@ -533,6 +536,38 @@ function (Okta, Q, Backbone, xdomain, SharedUtil, OktaAuth, Util, Expect, Router
       });
     });
 
+    describe('Events', function () {
+      itp('triggers a pageRendered event when first controller is loaded', function() {
+        return setup()
+        .then(function (test) {
+          test.router.primaryAuth();
+          return tick(test)
```

Comment:
Since we're working with Promises, it's nicer to just return the promise and not have the indent, i.e. [here](https://github.com/anishashetty-okta/okta-signin-widget/blob/as-OKTA-87152/test/spec/LoginRouter_spec.js#L226).
```javascript
// good
return someP(function (test) {
  return tick(test);
})
.then(function (test) {
  // continue on here
});

// not good
return someP(function (test) {
  return tick(test)
  .then(function () {
    // not so bad with one promise level, but gets much worse later
  });
})


```